src/v2/Apps/Home/Components/HomeFeaturedMarketNews.tsx

```diff
@@ -82,8 +91,10 @@ const HomeFeaturedMarketNews: React.FC<HomeFeaturedMarketNewsProps> = ({
             <Text variant="xs" textTransform="uppercase" my={1}>
               {firstArticle.vertical}
             </Text>
-
+          
+          <StyledTitle to={firstArticle.href ?? ""}>
```

Comment:
So right here you'd have an anchor tag nested within another anchor tag, which is invalid.

Reply:
What do you pass in here for the `to` prop then? It will not allow it to exist without something being passed for that prop?

Reply:
See above — you're won't be using a link here at all. There can only be a single link; which is the parent `RouterLink`